installer/installer.py

```diff
@@ -542,10 +544,14 @@ def setup_cosmovisor(self):
             self.remove_safe("CHANGELOG.md")
             self.remove_safe("README.md")
             self.remove_safe("LICENSE")
-            self.mkdir_p(self.cosmovisor_root_dir)
-            self.mkdir_p(os.path.join(self.cosmovisor_root_dir, "genesis"))
-            self.mkdir_p(os.path.join(self.cosmovisor_root_dir, "genesis/bin"))
-            self.mkdir_p(os.path.join(self.cosmovisor_root_dir, "upgrades"))
+            
+            if not is_cosmovisor_bump_needed:
+                self.mkdir_p(self.cosmovisor_root_dir)
+                self.mkdir_p(os.path.join(self.cosmovisor_root_dir, "genesis"))
+                self.mkdir_p(os.path.join(self.cosmovisor_root_dir, "genesis/bin"))
+                self.mkdir_p(os.path.join(self.cosmovisor_root_dir, "upgrades"))
+            else:
+                self.log(f"Bumping Comsovisor binary to {DEFAULT_LATEST_COSMOVISOR_VERSION}")
             if not os.path.exists(os.path.join(DEFAULT_INSTALL_PATH, DEFAULT_COSMOVISOR_BINARY_NAME)):
```

Comment:
If Cosmovisor is already installed at v1.1.0 and the user has asked to bump it, this logic will never be executed (if I'm right)